Python/Product/PythonTools/PythonTools/Navigation/CodeWindowManager.cs

```diff
@@ -143,8 +145,8 @@ class CodeWindowManager : IVsCodeWindowManager, IVsCodeWindowEvents {
             var wpfTextView = VsEditorAdaptersFactoryService.GetWpfTextView(vsTextView);
             if (wpfTextView != null) {
                 var factory = ComponentModel.GetService<IEditorOperationsFactoryService>();
-                var editFilter = new EditFilter(wpfTextView, factory.GetEditorOperations(wpfTextView), _serviceProvider);
-                editFilter.AttachKeyboardFilter(vsTextView);
+                EditFilter.GetOrCreate(_serviceProvider, ComponentModel, vsTextView);
+                ReplEditFilter.GetOrCreate(_serviceProvider, ComponentModel, vsTextView);
```

Comment:
Why do we deal with the REPL here?  The code window manager is dealing with windows which are normal code files hosted in the editor, and in particular is handling the navigation tool bar.  This code path shouldn't be hit at all for a REPL window